api/client.go

```diff
@@ -1189,6 +1219,11 @@ func (c *Client) clone(cloneHeaders bool) (*Client, error) {
 		CloneToken:     config.CloneToken,
 		ReadYourWrites: config.ReadYourWrites,
 	}
+
+	if config.CloneTLSConfig {
+		newConfig.clientTLSConfig = c.config.clientTLSConfig
+	}
```

Comment:
Are there cases where it's not desirable to have TLS config be cloned? Maybe we can just clone it by default (without a flag check).

Reply:
I would imagine so, but I wouldn't feel comfortable making a behavioral change like that as it might catch a user off guard.

Reply:
Fair enough 👍